examples/ovirt_repositories.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Setup repositories using oVirt release package
+  hosts: localhost
+
+  vars:
+    ovirt_reposotories_ovirt_release_rpm: http://resources.ovirt.org/pub/yum-repo/ovirt-master-release.rpm
+
+  roles:
+    - role: oVirt.repositories
+
+- vars_files:
+    # Contains encrypted `username` and `password` variables using ansible-vault
+    - passwords.yml
+
+- name: Setup repositories using Subscription Manager
```

Comment:
Would it be possible to split this example file? The 1st one for release package installation and the 2nd one for subscription manager